workflows/VGP-assembly-v2/Pre-curation-Hi-C-map-generation/PretextMap_Generation-tests.yml

```diff
@@ -8,14 +8,14 @@
       class: File
       location: https://zenodo.org/records/14218240/files/Haplotype%202.fasta
       filetype: fasta
-    HiC reads - forward:
+    Collection of HiC reads - forward:
```

Comment:
That seems unnecessary, the interface will say that it is a collection.

Reply:
I've had the question several times, people who didn't realise it needed to be in collection. I prefer to be obvious about it.